### ISSUE TITLE
Refactor badge.displayBrowserActionBadge to be used in both popup.js and messageHandler.js

### ISSUE THREAD

**Issue by groovecoder** [mozilla/multi-account-containers#973]
See https://github.com/mozilla/multi-account-containers/pull/921#issuecomment-341670787 for details.

**Comment by hritvi** [comment]
May I work on this if it is still open? Thanks :slightly_smiling_face:

**Comment by groovecoder** [comment]
Yes, you're welcome to work on it if you like. I haven't had time to triage or review many PRs though, so if you submit a PR see if you can make it small/concise? Thanks!

**Comment by hritvi** [comment]
While doing the initial setup using npm. A warning occurred:
<code>
found 20 vulnerabilities (3 low, 14 moderate, 2 high, 1 critical)
</code>
I suppose many packages need to be upgraded.
May I file a new issue for this and fix it?

**Comment by Aishat-Akinyemi** [comment]
Hello @groovecoder  I am a 2019 Outreachy applicant, May I work on this as my first contribution?

**Comment by groovecoder** [comment]
Hi @Aishat-Akinyemi - yes you can pick this one up.

**Comment by groovecoder** [comment]
I'm assigning this to myself to mark it as being assigned. Please let me know if you decide to work on a different issue.

**Comment by Aishat-Akinyemi** [comment]
I will start working on it, thank you.

**Comment by Aishat-Akinyemi** [comment]
I have submitted a Pull request that fixes this issue [https://github.com/mozilla/multi-account-containers/pull/1523#issue-326806982]

**Comment by maxxcrawford** [comment]
@Aishat-Akinyemi Can you [open up a PR](https://help.github.com/en/articles/creating-a-pull-request) for us to review?

**Comment by Aishat-Akinyemi** [comment]
@

> 
> 
> @Aishat-Akinyemi Can you [open up a PR](https://help.github.com/en/articles/creating-a-pull-request) for us to review?

@maxxcrawford  I already opened up a PR https://github.com/mozilla/multi-account-containers/pull/1523 that fixes this issue and issue #1513 . You commented on it saying I should make some changes. I have made those changes as requested, kindly check the PR  #1523

**Comment by maxxcrawford** [comment]
@Aishat-Akinyemi Nope – This works! I'll review. Thank you! ☺️
